migrations/1_initialize_schema.up.sql

```diff
@@ -369,6 +369,7 @@ CREATE TABLE dashboard_widget
   widget_id         BIGINT REFERENCES widget (id) ON DELETE CASCADE,
   widget_name       VARCHAR NOT NULL,
   widget_owner      VARCHAR NOT NULL,
+  widget_type       VARCHAR NOT NULL,
```

Comment:
Maybe we can remove from widget table similar columns such as widget_type and widget_name and leave them only in dashboard_widget.
Check if it is possible or not